pcs/lib/cib/acl.py

```diff
@@ -229,9 +232,10 @@ def create_target(acl_section, target_id):
     """
     # id of element acl_target is not type ID in CIB ACL schema so we don't need
     # to check if it is unique ID in whole CIB
-    if (
-        acl_section.find("./{0}[@id='{1}']".format(TAG_TARGET, target_id))
-        is not None
+    if acl_section.xpath(
+        "./*[local-name()=$target_tag and @id=$target_id]",
+        target_tag=TAG_TARGET,
+        target_id=target_id,
```

Comment:
Based on the comment about inserting constants into a xpath expresion, this could be changed like so:
```suggestion
    if acl_section.xpath(
        f"./{TAG_TARGET}[@id=$target_id]",
        target_id=target_id,
```

Reply:
ack